data/yam/agama/auto/sle_ext4.jsonnet

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
let's call it: `sle_root_filesystem_ext4` and same for the test suite name to configure in job group.
also same for the xfs file in this PR.